README.md

```diff
@@ -91,6 +91,10 @@ client.getInfo().then(([body, headers]) => console.log(body, headers));
 const [body, headers] = await client.getInfo();
 ```
 
+### Floating point number precision in JavaScript
+
+Due to [Javascript floating point precision](http://floating-point-gui.de/), to prevent precision loss when using `bitcoin-core` we should pass all big numbers (numbers with more than 15 significant digits) as string instead of using `Number` data type.
```

Comment:
Due to [JavaScript's limited floating point precision](http://floating-point-gui.de/), all *big* numbers (numbers with more than 15 significant digits) are returned as strings to prevent precision loss.